### ISSUE TITLE
Unable to run on server

### ISSUE THREAD

**Issue by spannerman79** [Silly511/DimensionalCore#2]
DimensionalCore-1.7.10-1.0.3.jar
ResonantToolbox-1.7.10-1.0.6.jar
ThermalExpansion-[1.7.10]4.1.2B1-239.jar
forge-1.7.10-10.13.4.1614-1.7.10-universal.jar

Error:

```
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/entity/Render
        at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
        at cpw.mods.fml.common.Loader.loadMods(Loader.java:544)
        at cpw.mods.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:87)
        at cpw.mods.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:314)
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/entity/Render
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:348)
        at cpw.mods.fml.common.FMLModContainer.constructMod(FMLModContainer.java:440)
        at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
        at com.google.common.eventbus.EventBus.post(EventBus.java:275)
        at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
        at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
        at com.google.common.eventbus.EventBus.post(EventBus.java:275)
        at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
        at cpw.mods.fml.common.Loader.loadMods(Loader.java:513)
        ... 5 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.entity.Render
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 29 more
Caused by: java.lang.ArrayIndexOutOfBoundsException
```

There is no client render on a server :frowning:

**Comment by Fadenfire** [comment]
I think I fixed it in Resonant Toolbox 1.0.7, but I'm not sure. Could you make sure everything works ok?

**Comment by spannerman79** [comment]
:+1: Fixed.

Apologies that this was created in the incorrect issue tracker.
